### ISSUE TITLE
Naming conflict with bullet from libbullet-dev

### ISSUE THREAD

**Issue by andreflorindo** [ros-industrial-consortium/tesseract_ext#37]
Hi, while trying to compile tesseract I was faced with the following compilation error:

```bash
/home/andreflorindo/workspaces/tesseract_ws/src/tesseract/tesseract_collision/src/bullet/tesseract_convex_convex_algorithm.cpp:
689:26: error: ‘class btConvexPolyhedron’ has no member named ‘initialize2’; did you mean ‘initialize’?
               polyhedron.initialize2();
```
For reference, the computer that I used had melodic and ubuntu 18.04. After a little digging, I identified that the bullet library that tesseract was using could be traced back to my `usr/include/bullet` folder, most likely created when installing `libbullet-dev`. For my setup, the last version of `libbullet-dev` available is 2.87 and `initialize2` is not part of it. 

This error is due to the change of name from `bullet3` to `bullet` in [one](https://github.com/ros-industrial-consortium/tesseract_ext/commit/3a991960e0119957717d12cbbc3fb65326ff60ed) of the last commits . Going back to the december commit this error did not show up. Or just removing `libbullet-dev` also solved the issue.

**Comment by Levi-Armstrong** [comment]
We actually ran into this issue yesterday. We have a PR coming which will check the version of bullet, but for now if you pass -DINSTALL_BULLET=ON it will force it to build bullet. I assume you are on 18.04?

**Comment by andreflorindo** [comment]
Your assumption is correct. And forcing bullet3 installation is fine, at least for the time being.

**Comment by Levi-Armstrong** [comment]
This should be resolved by #38.
